src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1411,6 +1412,23 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
             return;
         }
 
+        const auto timeNow = std::chrono::high_resolution_clock::now();
+
+        if (_lastScrollTime.has_value())
+        {
+            // Limit update operation to frame rate
+            // See also: Microsoft::Console::Render::RenderThread::s_FrameLimitMilliseconds
+            static constexpr long long s_ScrollRateLimitMilliseconds = 8;

```

Comment:
We usually place constants like this at the top of the CPP file or in a header file.